benchmarks/perf-tool/release-configs/lucene-hnsw/relaxed-filter-test.yml

```diff
@@ -0,0 +1,33 @@
+endpoint: [ENDPOINT]
+test_name: "index-workflow"
+test_id: "Index workflow"
+num_runs: 10
+show_runs: false
+steps:
+  - name: delete_index
+    index_name: target_index
+  - name: create_index
+    index_name: target_index
+    index_spec: /home/ec2-user/[PATH]/index.json
```

Comment:
```suggestion
    index_spec: [INDEX_SPEC_PATH]/index.json
```

Reply:
I think `/home/ec2-user` shouldn't be specified here.

Reply:
ack